src/transformation/visitors/class/index.ts

```diff
@@ -349,3 +346,15 @@ export const transformSuperExpression: FunctionVisitor<ts.SuperExpression> = (ex
 };
 
 export const transformThisExpression: FunctionVisitor<ts.ThisExpression> = node => createSelfIdentifier(node);
+
+function getReflectionClassName(context: TransformationContext, declaration: ts.ClassLikeDeclaration): string {
+    if (declaration.name) {
+        return declaration.name.text;
+    } else if (ts.isVariableDeclaration(declaration.parent) && ts.isIdentifier(declaration.parent.name)) {
+        return declaration.parent.name.text;
+    } else if (hasDefaultExportModifier(declaration)) {
+        return context.createUniqueString("default");
```

Comment:
Isn't it just `"default"`?

Reply:
In my test JS produced 'default_1', although I am still a bit hesitant on this because I'm pretty sure we will deviate for multiple anyway (since numberings will be different).

Reply:
You have probably tested some transpiled version, in native ESM on node it's just `"default"`